src/ng-select/lib/ng-select.component.ts

```diff
@@ -414,7 +414,7 @@ export class NgSelectComponent implements OnDestroy, OnChanges, OnInit, AfterVie
     }
 
     open() {
-        if (this.disabled || this.isOpen || this.itemsList.maxItemsSelected || this._manualOpen) {
```

Comment:
I don't see any test changes here. Also, the build check is missing in PR. Are those tests are passing?